src/sink/elastic.rs

```diff
@@ -241,13 +241,36 @@ fn build_event_payload(event: &Event) -> Result<Vec<u8>> {
         if let Some(pipeline) = meta.get_str("pipeline") {
             index_meta.insert("pipeline", pipeline)?;
         };
-        let value_meta = json!({ "index": index_meta });
+        let key = match meta.get_str("es_op") {
```

Comment:
i would call it `action` as this is the elasticsearch terminology here: https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html#bulk-update

We might want to namespace those metadata values under `elastic` as we do with other sinks, to avoid potential collisions with other `action` metadata values.

Reply:
Good call! I hated the name :D `asction` is much better!